web/static/dlg_repo_edit.js

```diff
@@ -188,7 +184,7 @@ export function show( a_repo_id, a_cb ){
                     node.data.alloc.totalSz = data.totalSz;
                     updateAllocTitle( node );
 
-                    var msg =
+                    var msg =a_repo_id
```

Comment:
So this looks like it is only sending the repo_id back is that the intention or is my javascript off?

Reply:
No, that's a weird typo. Not sure how that got there but needs to be fixed (there's a msg body below this not shown).